src/styles/main.scss

```diff
@@ -1,7 +1,582 @@
-@import "utils";
 @import "fonts";
+@import "normalize";
+@import "utils";
 @import "typography";
 
+* {
```

Comment:
>19	Remember to reset default margins for headings (h1, h2, h3, etc.).

Reply:
`margin: 0` resets default size of margins, if I use it inside of the `*` for headings, my all headings margins I wrote will be broken.